rpc/jsonrpc/server/http_json_handler.go

```diff
@@ -54,6 +55,29 @@ func makeJSONRPCHandler(funcMap map[string]*RPCFunc, logger log.Logger) http.Han
 			}
 			requests = []types.RPCRequest{request}
 		}
+		// read the maxBatchRequestNum from header
+		maxBatchRequestNum, err := strconv.Atoi(r.Header.Get("MaxBatchRequestNum"))
```

Comment:
Is header's key case CamelCase in jsonrpc? I think general header key case is like `Xxx-Yyy-Xxx`.

Reply:
BTW, is `MaxBatchRequestNum` value received from the request header?

Reply:
After the server receives the request, a handler will add this value to the request header.
https://github.com/Finschia/ostracon/pull/672/files#diff-7ddabaeab69b424ea8224aa78312290461a2e51f69da68bb7bd7532d51fb1a5cR290-R300
```
func (h maxBatchRequestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
	r.Header.Set(h.NewHeaderName, h.NewHeaderValue)
	h.h.ServeHTTP(w, r)
}
```

Reply:
Since the http_json_handler function does not use configuration, adding this field will result in many function modifications.